junit_xml/__init__.py

```diff
@@ -291,9 +318,10 @@ def _clean_illegal_xml_chars(string_to_clean):
 class TestCase(object):
     """A JUnit test case with a result and possibly some stdout or stderr"""
 
-    def __init__(self, name, assertions=None, elapsed_sec=None,
-            timestamp=None, classname=None, status=None, category=None, file=None, line=None,
-            log=None, group=None, url=None, stdout=None, stderr=None):
+    def __init__(self, name, classname=None, elapsed_sec=None, stdout=None,
```

Comment:
I changed back the signature as it was in #38  before my  #41 (where I broke the signature)
`     def __init__(self, name, classname=None, elapsed_sec=None, stdout=None,
stderr=None):`